src/config/file/v1.rs

```diff
@@ -149,9 +175,27 @@ impl TryFrom<Integration> for remote::Integration {
                 team,
                 title,
                 auth,
+                import_branches,
+                import_tags,
+                watched_branches,
             } => {
                 let poll_interval = remote::PollInterval::try_from(poll_interval)?;
                 let endpoint = remote::Remote::try_from(remote)?;
+                let import_branches = import_branches.unwrap_or(true);
+                let import_tags = import_tags.unwrap_or(false);
+                let watched_branches = watched_branches
+                    .unwrap_or_default()
+                    .into_iter()
+                    .map(remote::WatchedBranch::new)
+                    .collect::<Vec<remote::WatchedBranch>>();
```

Comment:
[nit, optional] you can infer LHS most of the time:
```suggestion
                    .collect::<Vec<_>>();
```

or use `itertools`' `collect_vec`.